marklogic-data-hub/src/testFixtures/java/com/marklogic/hub/test/AbstractHubTest.java

```diff
@@ -207,6 +207,11 @@ protected void setTestUserRoles(String... roles) {
         user.save();
     }
 
+    protected int getDocumentCount(DatabaseClient client) {
```

Comment:
Can you put this at the bottom of the file? Will help avoid merge conflicts (I'm making changes to this file too)